scene/gui/control.cpp

```diff
@@ -46,6 +46,8 @@
 #include "servers/rendering_server.h"
 #include "servers/text_server.h"
 
+#define ARRAY_SIZE(a) (sizeof(a) / sizeof(a[0]))
```

Comment:
Instead of defining a macro, you could add a constant (which you could then use in `properties_managed_by_container` instead of using magic 11)